openstack/intel-worker/playbooks/roles/centosImage-create/tasks/main.yml

```diff
@@ -1,27 +1,21 @@
 ---
-- name: OpenStack worker node configuration Tasks (i.e. Create worker.ign, Upload RHCOS Image, Create Customization Script, Create Instance etc)
-  hosts: all
-  vars_files:
-    - ../../../../vars/main.yml
+- name: Download the Image from the URL and use to create image
```

Comment:
What do you think about kebab case so we follow a consistent role? 
`centosImage-create` we should change to `centos-image-create`

we'd update rhcos too. I think we can leave this for the next PR (if it makes sense)

Reply:
yes , I thought to have `centos-image-create` , but ansible-lint was giving me error for this name with twice `-` in the name , so because of this , thought to name it as `centosImage-create` .